### DISCUSSION TITLE
How does orientation get update during plastic deformation for a material point?

### DISCUSSION THREAD

**Discussion by vivy234** [neperfepx/FEPX#45] [Q&A] [answered]
Hi, 
I want to understand the formulation of the updated orientation for a material point after a time increment. Could you please suggest me?
Thank you.
With regards,
Vivek

**Comment by mkasemer** [comment, marked as answer]
@vivy234 

The theory behind the implementation is described [here](https://arxiv.org/abs/1504.03296), and in specific what you seek is in S4.2.3.

**Comment by vivy234** [comment]
Thank you.

On Wed, Sep 29, 2021 at 11:49 PM Matthew Kasemer ***@***.***>
wrote:

> @vivy234 <https://github.com/vivy234>
>
> The theory behind the implementation is described here
> <https://arxiv.org/abs/1504.03296>, and in specific what you seek is in
> S4.2.3.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/acmelab-ua/FEPX/discussions/45#discussioncomment-1404702>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOQIOSCEF2GYOVICSDNPZI3UENKDBANCNFSM5FAOXXHQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
With regards,
Vivek Kumar Sahu
Research Scholar
Texture-Microstructure-Stress Lab
IIT Kanpur, Kanpur
208016, India
Google Scholar link:
https://scholar.google.com/citations?user=OGc7M14AAAAJ&hl=en
